### PR TITLE
Supersede mkdirp with fs-extra.

### PR DESCRIPTION
**Pull request by hogofwar**
Implements #380.

### CODE REVIEW

File: src/command-line/index.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ var argv = program.parseOptions(process.argv);
 Helper.setHome(program.home);
 
 if (!fs.existsSync(Helper.CONFIG_PATH)) {
-	mkdirp.sync(Helper.HOME, {mode: "0700"});
+	fsextra.ensureDirSync(Helper.HOME);
+	fs.chmod(Helper.HOME, 0700)
```
Comment (dgw) [inline src/command-line/index.js]: missing `;`, and `0700` should be quoted to avoid the other eslint error about octal literals.

**Comment by dgw** [comment]
Not sure why this was closed; the failed checks are just due to a linter failure, which is easily fixed (see line notes).

**Comment by hogofwar** [comment]
Yeah, I prematurely closed it. Accidentally removed mkdirp from mocha, which made me think there was a bigger problem. During fixing that found those issues you mentioned and fixed them.

Then I figured out how to squash commits so now I unfortunately have to create a new pull request.

**Comment by dgw** [comment]
> Then I figured out how to squash commits so now I unfortunately have to create a new pull request.

No, you usually just have to `git push -f` to the branch you opened the PR from and GitHub will update the PR.

**Comment by hogofwar** [comment]
Yep, I did that! But apparently since this was closed already it won't allow me to reopen it when the branch it was taken from was rebased.

**Comment by dgw** [comment]
Well that's good to know for my own future GH use. Note to self: Don't force push to a closed PR's branch if you want to reopen it. Reopen, then force push.

Good lesson for me, at your expense, haha.

**Comment by hogofwar** [comment]
If someone learns from my mistakes, that's great!

**Pull request closed without merging**